externs/olx.js

```diff
@@ -7327,6 +7327,15 @@ olx.source.ZoomifyOptions.prototype.transition;
 
 
 /**
+ * Tile size. Same tile size is used for all zoom levels. Default value is
+ * `OpenLayers.DEFAULT_TILE_SIZE`.
+ * @type {number|undefined}
+ * @api
+ */
+olx.source.ZoomifyOptions.prototype.tileSize;
```

Comment:
`tileSize` also needs to be added to `olx.source.ZoomifyOptions;` above.

Reply:
I have now fixed this.